diff -urN cairomm-1.8.4/configure.ac cairomm-1.8.4.new/configure.ac
--- cairomm-1.8.4/configure.ac	2009-10-19 16:01:52.000000000 +0200
+++ cairomm-1.8.4.new/configure.ac	2010-02-09 00:52:22.179852265 +0100
@@ -43,7 +43,7 @@
 cairomm_allmodules=$CAIROMM_MODULES
 CAIROMM_INSTALL_PC='data/cairomm-1.0.pc'
 
-for cairomm_mod in ft pdf png ps svg xlib xlib-render \
+for cairomm_mod in ft pdf png ps svg xlib xlib-xrender \
                    win32 win32-font quartz quartz-font quartz-image
 do
   PKG_CHECK_EXISTS([cairo-$cairomm_mod],
