--- cairomm-0.5.0/cairomm/surface.cc.orig	2006-01-29 06:00:31.000000000 +0100
+++ cairomm-0.5.0/cairomm/surface.cc	2006-02-11 00:12:03.724417250 +0100
@@ -288,7 +288,7 @@
   // surface is destroyed in base class
 }
 
-RefPtr<GlitzSurface> GlitzSurface::create(glitz_surface_t *surface))
+RefPtr<GlitzSurface> GlitzSurface::create(glitz_surface_t *surface)
 {
   cairo_surface_t* cobject = cairo_glitz_surface_create(surface);
   check_status_and_throw_exception(cairo_surface_status(cobject));
